external/fv3fit/tests/emulation/layers/test_normalization.py

```diff
@@ -12,11 +12,25 @@
 
 def test_norm_layer():
     norm = NormLayer(scale=2.0, center=1.0)
-    x = tf.ones([1], dtype=tf.float32)
+    x = tf.ones([1])
```

Comment:
Did you remove this because float32 breaks the test, or because this is no longer needed?

Reply:
This was no longer needed.